transfers/file-transfers.md

```diff
@@ -8,28 +8,39 @@ parent: Transfers
 # File Transfers
 {: .no_toc }
 
-## Table of contents
+<!--## Table of contents
 {: .no_toc .text-delta }
 
 1. TOC
-{:toc}
+{:toc}-->
 
-# Introduction
+<!--# Introduction
 
 Born digital collection material can be acquired through file transfer or forensic imaging. Most material will be transferred using a Bagit script through command line.
 
-The file transfer workflows are detailed in this document. The workflows may vary based on media types and file types.  
+The file transfer workflows are detailed in this document. The workflows may vary based on media types and file types-->
 
-**Before a media object can be transferred it first must be recorded SPEC.**
+This page describes instructions for acquiring born digital material via file transfer and packaging file transfers for processing. The file transfer workflow described on this page primarily applies to internal and external hard drives, thumb drives and sd cards.
 
-## Prepare destination folders for files
+### Before Imaging
+{: .important }
+Before any media can be imaged **it first must have an object record in SPEC**.
```

Comment:
```suggestion
Before any media can be imaged **it first must have an object record in SPEC**.
```
Before any media can be transferred **it first must have an object record in SPEC**.